effekt/jvm/src/test/scala/effekt/core/RenamerTests.scala

```diff
@@ -17,6 +19,52 @@ class RenamerTests extends CoreTests {
     assertAlphaEquivalent(obtained, pInput, clue)
   }
 
+  def assertRenamingMakesDefsUnique(input: String,
+                                    clue: => Any = "Duplicate definition",
+                                    names: Names = Names(defaultNames))(using munit.Location) = {
+    val pInput = parse(input, "input", names)
+    val renamer = new Renamer(names, "renamed")
+    val obtained = renamer(pInput)
+
+    val seen = mutable.HashSet.empty[Id]
+    def isFresh(id: Id): Unit = {
+      assert(!seen.contains(id), clue)
```

Comment:
Something like the following might help with the error message, I guess?
```suggestion
                                    clue: Id => Any = id => s"Duplicate definition of ${id}",
                                    names: Names = Names(defaultNames))(using munit.Location) = {
    val pInput = parse(input, "input", names)
    val renamer = new Renamer(names, "renamed")
    val obtained = renamer(pInput)

    val seen = mutable.HashSet.empty[Id]
    def isFresh(id: Id): Unit = {
      assert(!seen.contains(id), clue(id))
```